### PR TITLE
Upgrade laravel exceptions to 8.1 - closes #1345

### LINKED ISSUE #1345: Upgrade to laravel exceptions 8.1 with config changes

### PR DESCRIPTION
**Pull request by jbrooksuk**

### CODE REVIEW

File: config/exceptions.php [hunk 1]
```diff
@@ -48,6 +49,7 @@
         'CachetHQ\Cachet\Foundation\Exceptions\Displayers\JsonValidationDisplayer',
         'CachetHQ\Cachet\Foundation\Exceptions\Displayers\RedirectDisplayer',
         'GrahamCampbell\Exceptions\Displayers\DebugDisplayer',
+        'GrahamCampbell\Exceptions\Displayers\ViewDisplayer',
```
Comment (GrahamCampbell) [inline config/exceptions.php]: This was intentionally left out wasn't it

File: config/exceptions.php [hunk 2]
```diff
@@ -100,8 +102,8 @@
     */
 
     'levels' => [
+        'Illuminate\Auth\Access\AuthorizationException'                 => 'warning',
         'Illuminate\Session\TokenMismatchException'                     => 'notice',
-        'Illuminate\Database\Eloquent\ModelNotFoundException'           => 'warning',
```
Comment (GrahamCampbell) [inline config/exceptions.php]: ?

Comment (jbrooksuk) [inline config/exceptions.php]: Whoops, forgot to copy this over.

File: config/exceptions.php (hunk 1 above)
Comment (jbrooksuk) [inline config/exceptions.php]: It may have been.

Comment (GrahamCampbell) [inline config/exceptions.php]: Yeh, we don't use it, so there's little point in making the code spin through it every time.

Comment (jbrooksuk) [inline config/exceptions.php]: I'll remove it.

**Comment by jbrooksuk** [comment]
Config updated.

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request merged**